spec/models/task_spec.rb

```diff
@@ -8,9 +8,17 @@
 
     let(:user) { nil }
     let(:task_user) { create(:user) }
-    let(:task) { create(:task, user: task_user) }
-    let(:private?) { false }
+    let(:group_user) { create(:user) }
+    let(:access_level) { 'private' }
+    let!(:group) { create(:group) }
```

Comment:
```suggestion
    let(:group_memberships) { [build(:group_membership, :with_admin), build(:group_membership, user: group_user, role:)] }
    let(:group) { create(:group, group_memberships:) }

```

and remove the `before`-block in line 18++